apps/catalogue/components/landing/CardPrimary.vue

```diff
@@ -41,7 +55,18 @@ withDefaults(defineProps<Props>(), {
       <span
         class="md:hidden absolute right-0 mr-3 hover:text-blue-800 text-blue-500"
       >
-        <NuxtLink :to="link" :target="openLinkInNewTab ? '_blank' : undefined">
+        <a
+          v-if="renderAsAnchorElem"
+          :href="link"
+          :target="openLinkInNewTab ? '_blank' : undefined"
+        >
+          <IconButton icon="arrow-right" />
+        </a>
+        <NuxtLink
+          v-else
+          :to="link"
+          :target="openLinkInNewTab ? '_blank' : undefined"
+        >
```

Comment:
I've updated this link, but I'm not sure why this one is here. There are already two links to the same page. What purpose does this link serve?